hardhat/hardhat.config.ts

```diff
@@ -66,12 +66,20 @@ task('create-user-policy', 'Creates a policy for the given user')
   .addParam('coverageAmount', 'The coverage amount')
   .addParam('ipfsHash', 'The IPFS policy hash')
   .setAction(async (args, hre) => {
+    const network = hre.network.name;
+    const deploymentFileName = `../src/contract-deployments/${network}-dao.json`;
```

Comment:
I followed the same approach here as the existing "sent-to-account" task (to handle the different networks). My question is though, should we be allowing these tasks to run on networks other than localhost? I can't imagine we would ever want to run this on `mainnet`